data/korath/korath missions.txt

```diff
@@ -257,3 +257,48 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Korath Boarding Response"
+	invisible
+	boarding
+	repeat
+	minor
+	deadline 1
+	to offer
+		not "Korath Boarding Response: active"
+		or
+			and
+				random < 5
+				"reputation: Korath" < -10
+			and
+				random < 10
+				"reputation: Korath" < -100
+			and
+				random < 25
+				"reputation: Korath" < -1000
+			and
+				random < 50
+				"reputation: Korath" < -10000
+
+			and
+				random < 10
+				"reputation: Korath (Civilian)" < 0
+			and
+				random < 25
+				"reputation: Korath (Civilian)" < -100
```

Comment:
Isnt this like just one worldship to kill?

Reply:
Yes. The Korath don't like it when you kill their civilians.

Reply:
I changed it to -300, which will probably work out to two worldships.

Reply:
Yeah caus otherwise it was the same as < 0